### PR TITLE
Fix logger test case, add SetSkipLine func.

### PR DESCRIPTION
**Pull request by Colstuwjx**
RT, fix bad test case.

### CODE REVIEW

**Comment by Colstuwjx** [comment]
Ping @reasonerjt

**Comment by ywk253100** [comment]
@Colstuwjx Can you explain why this PR is needed?

**Comment by Colstuwjx** [comment]
I'm learning from harbor sourcecode, and I found there is a logical mistake in test case, so I opened PR to fix it, that' it.

**Comment by ywk253100** [comment]
What logic mistake? Which line?

**Comment by Colstuwjx** [comment]
Like
 
```
if strings.HasSuffix(str, "[ERROR] message") {
    ..
}
```

due to the default logger set skipLine `false`, the message would be

```
2018-09-04T10:50:27+08:00 [ERROR] [logger_test.go:124]: message
```

Of cause it wouldn't be true for `if strings.HasSuffix(str, "[ERROR] message")` this condition
But I don't think it tested the logger functions.

So I raised this PR.
Thanks.

File: src/common/utils/log/logger.go [hunk 1]
```diff
@@ -93,6 +93,14 @@ func (l *Logger) SetLevel(lvl Level) {
 	l.lvl = lvl
 }
 
+//SetSkipLine sets the skipLine of Logger l
+func (l *Logger) SetSkipLine(skipLine bool) {
```
Comment (ywk253100) [inline src/common/utils/log/logger.go]: The `skipLine` is an internal variable, we shouldn't expose it. 
Even though we call the method to set `skipLine` as `true`, the `Debug()`, `Info()` and `Warning()` don't print the information about line either.

**Comment by ywk253100** [comment]
@Colstuwjx Got it, thanks.

File: src/common/utils/log/logger.go (hunk 1 above)
Comment (ywk253100) [reply src/common/utils/log/logger.go]: For 1, the `skipLine` is in the same scope with the test codes, you can just set it to `true` by calling `logger.skipLine = true` directly. 
And I think maybe we shouldn't set it to `true`, we should test it for the real usage.
For 2, I don't think it's the issue that Harbor should fix. The log collector shouldn't require that all the applications are using the same log format.

Comment (Colstuwjx) [reply src/common/utils/log/logger.go]: Fixed, thanks.

**Comment by ywk253100** [comment]
@Colstuwjx Thanks a lot.

**Pull request merged**